.github/workflows/build.yml

```diff
@@ -30,9 +34,7 @@ jobs:
         hss1394
         libebur128
         libflac
-        libid3tag
         libkeyfinder
-        libmad
```

Comment:
Are these not needed for macOS because we use CoreAudio? Ping @uklotzde

Reply:
I copied the libid3tag for Tenacity but it does not build for macOS.

Reply:
Yes, it is not used in MacOS.

Reply:
okay